probes/sbomExists/def.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: sbomExists
+short: Check that the project has an sbom file
+motivation: >
+  A sbom can give users information about how the source code components and dependencies. They help facilitate sotware supplychain security and aid in identifying upstream vulnerabilities in a codebase.
+implementation: >
+  The implementation checks whether an sbom file is present in the source code.
```

Comment:
So I wanted this one to award points if any sbom was found, source or otherwise. So that release/CI boms would get full points but source only gets partial.